### PR TITLE
Declare connection in multiple bindings addon.xml

### PR DESCRIPTION
**Pull request by mlobstein**
Declare connection type in addon.xml for bindings where I am listed as code owner.

### CODE REVIEW

**Review by wborn — COMMENTED** [review]
Did you also check/update the translation files for these `addon.xml` changes?

File: bundles/org.openhab.binding.atlona/src/main/resources/OH-INF/addon/addon.xml [hunk 1]
```diff
@@ -4,6 +4,8 @@
 	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
 
 	<type>binding</type>
-	<name>Atlona Products</name>
+	<name>Atlona Binding</name>
```
Comment (jlaur) [inline bundles/org.openhab.binding.atlona/src/main/resources/OH-INF/addon/addon.xml:7]: Verified:
https://github.com/openhab/openhab-addons/blob/b6da1ec590833063d21909b58170227b01521914/bundles/org.openhab.binding.atlona/src/main/resources/OH-INF/i18n/atlona.properties#L3

File: bundles/org.openhab.binding.kaleidescape/src/main/resources/OH-INF/addon/addon.xml [hunk 2]
```diff
@@ -4,7 +4,8 @@
 	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
 
 	<type>binding</type>
-	<name>Kaleidescape System Binding</name>
-	<description>Controls a Kaleidescape System Movie Player</description>
+	<name>Kaleidescape Binding</name>
```
Comment (jlaur) [inline bundles/org.openhab.binding.kaleidescape/src/main/resources/OH-INF/addon/addon.xml:7]: Verified:
https://github.com/openhab/openhab-addons/blob/b6da1ec590833063d21909b58170227b01521914/bundles/org.openhab.binding.kaleidescape/src/main/resources/OH-INF/i18n/kaleidescape.properties#L3

**Review by jlaur — APPROVED** [review]
Thanks!

**Comment by mlobstein** [comment]
> Did you also check/update the Atlona/Kaleidescape translation files for these `addon.xml` changes?

These were changed in #14080 and #13593 but the migration to addon.xml (#13289) drifted back to the previous values.

**Pull request merged**